### ISSUE TITLE
feat: return number of failed rules and checks

### ISSUE THREAD

**Issue by JosephBARBIERDARNAL** [JosephBARBIERDARNAL/page#306]
A rule is unique, while a check is just one time the rule failed. Example: if 3 images don't have alt text in UA/1, then this leads to +1 failed ruled (missing alt text) and +3 failed checks.

- CLI: add a ligne after Conformant/Non-conformat with `17 failed rules | 132 failed checks` (or something else similar?). Add to JSON output too.
- Rust/Python/JS: add as attributes of the `ValidationReport`
